installers/rpm-code/codes/aravis.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+aravis_main() {
+    # Not visible in GH tabs: https://github.com/AravisProject/aravis/releases
+    codes_yum_dependencies \
+        cmake \
+        g++ \
+        gettext \
+        glib2-devel \
+        gobject-introspection \
+        gobject-introspection-devel \
+        gstreamer1-devel \
+        gstreamer1-plugins-base-devel \
+        gstreamer1-plugins-good \
+        gtk-doc \
+        gtk3-devel \
+        libusb1-devel \
+        libxml2-devel \
+        libxslt \
+        meson \
+        python-gobject
+    codes_dependencies common
+    codes_download https://github.com/AravisProject/aravis/releases/download/0.8.31/aravis-0.8.31.tar.xz
+    meson setup \
+        --prefix="${codes_dir[prefix]}" \
+        --libdir="${codes_dir[prefix]}"/lib build
+    cd build
+    ninja -j"$(codes_num_cores)"
+    ninja install
```

Comment:
FWIW - I was curious about this. The meson docs linked to https://neugierig.org/software/chromium/notes/2011/02/ninja.html

I know some of our other code(s?) have the option to use ninja but I've turned it off.

Reply:
neugierig.org got me curious. Went down a long rabbit hole. The number of people out there who are amazing is, well, amazing. Thanks.

I had no idea what ninja is or why. I just followed the instructions.